lib/testsuite/index.js

```diff
@@ -213,7 +213,8 @@ class TestSuite {
         modulePath,
         reportPrefix: '',
         reportFileName: this.argv['report-filename'],
-        groupName
+        groupName,
+        isMobile: this.client.api.isMobile()
```

Comment:
I think we used `isMobile` from utils over here instead of calling an API method

Reply:
Something like this? `isMobile(this.settings.desiredCapabilities)`

But I think calling api method is better because that will mean we have only one place to update the behaviour if something changes. Also this API method does not queue a task, its a normal function call, hence I thought it does not make a difference.

Reply:
Yes but I think for future proof we shouldn't call API methods over here.

Reply:
This will require too much change for this PR. Will refactor later.